src/test/java/com/revature/controllertests/ContentControllerTest.java

```diff
@@ -30,99 +27,147 @@
+import com.revature.entities.Link;
+import com.revature.services.ContentService;
+import com.revature.services.SearchService;
+import com.revature.testingutils.ContentFactory;
 
 @SpringBootTest(classes = CMSforceApplication.class)
 public class ContentControllerTest extends AbstractTestNGSpringContextTests {
 
-	//allows us to send mocked http requests
+	// allows us to send mocked http requests
 	private MockMvc mvc;
 
-	//allows json<->object conversion
-    private ObjectMapper objMapper = new ObjectMapper();
+	// allows json<->object conversion
+	private ObjectMapper objMapper = new ObjectMapper();
 
-	//controller being tested
+	// controller being tested
 	@InjectMocks
 	private ContentController cc;
-	
-	//service used by controller
+
+	// service used by controller
 	@Mock
 	private ContentService cs;
+
+	@Mock
+	private SearchService ss;
 	
-	//content being passed to controller requests
+	// content being passed to controller requests
 	private Content content;
-	
+
 	/**
-	 * Initialize Mockito and mocking dependencies 
+	 * Initialize Mockito and mocking dependencies
 	 */
 	@BeforeClass
-	public void setup () {
-		//build mock MVC so can build mock requests
-		cc = new ContentController ();
+	public void setup() {
+		// build mock MVC so can build mock requests
+		cc = new ContentController();
 		mvc = MockMvcBuilders.standaloneSetup(cc).build();
-		
-		//enables mockito annotations
+
+		// enables mockito annotations
 		MockitoAnnotations.initMocks(this);
 	}
-	
+
 	/**
 	 * Ensure clean content for each test
 	 */
 	@BeforeTest
-	public void preTestSetup () {
+	public void preTestSetup() {
 		content = ContentFactory.getContent();
 	}
-	
+
 	/**
 	 * Test adding new content to the back-end
+	 * 
 	 * @throws Exception - if mocked http request fails
 	 */
 	@Test
-	public void givenValidDataCreateContent () throws Exception {
-		//given
+	public void givenValidDataCreateContent() throws Exception {
+		// given
 		Mockito.when(cs.createContent(content)).thenReturn(content);
-		
-		//when
-		ResultActions result = mvc.perform(post ("/content")
-								.contentType(MediaType.APPLICATION_JSON_VALUE)
-								.content(objMapper.writeValueAsString(content)));
-		Content ret = objMapper.readValue(result.andReturn().getResponse()
-						  .getContentAsString(), Content.class);
-		
-		//then
-		//expect status of OK
+
+		// when
+		ResultActions result = mvc.perform(post("/content").contentType(MediaType.APPLICATION_JSON_VALUE)
+				.content(objMapper.writeValueAsString(content)));
+		Content ret = objMapper.readValue(result.andReturn().getResponse().getContentAsString(), Content.class);
+
+		// then
+		// expect status of OK
 		result.andExpect(status().isOk());
-		//content should be created
-		assertEquals (ret, content, "Failed to create content");
+		// content should be created
+		assertEquals(ret, content, "Failed to create content");
 	}
-	
+
+	@Test
+	public void givenValidDataCreateContent_GetLinksNotNull() throws Exception {
```

Comment:
method naming conventions; shouldn't include an underscore

Reply:
Okay, I will change that. I saw a few conventions so I simply chose that one. I will go to strictly camelcase.

Reply:
relevant in areas like src/test/java/com/revature/entities/ModuleTest.java as well